spine/model/layer/cnn/ppn.py

```diff
@@ -553,14 +629,23 @@ def forward(self, ppn_label, ppn_points, ppn_masks, ppn_layers, ppn_coords,
             # Compute the pairwise distances between each label point
             # and all the voxels in the image.
```

Comment:
Move this comment inside the `else`, as it's where the distance is computed. Add another relevant comment here about dispatching.

Reply:
Done.